src/OrchardCore.Modules/OrchardCore.Spatial/Views/GeoPointField.cshtml

```diff
@@ -0,0 +1,11 @@
+@model DisplayGeoPointFieldViewModel
+@using OrchardCore.Mvc.Utilities;
+@using OrchardCore.Spatial.ViewModels
+
+@{
+    var name = (Model.PartFieldDefinition.PartDefinition.Name + "-" + Model.PartFieldDefinition.Name).HtmlClassify();
+}
+
+<div class="field field-type-geopointfield field-name-@name">
+    @String.Format("{0:0.000000}", Model.Field.Latitude), @String.Format("{0:0.000000}", Model.Field.Longitude)
```

Comment:
Would it use `,` in French for the decimal separator, or is the `.` part of a standard

Reply:
It will use a `,` in french so I think we could/should probably set a different placeholder value simply by culture.

Reply:
Here same thing we need to check for a field setting to adjust the UI consequently with the field maxlength precision.